### PR TITLE
Ganache cli

### PR DESCRIPTION
**Pull request by benjamincburns**
Changes to switch everything over to ganache-cli.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,36 @@
-[![npm](https://img.shields.io/npm/v/ethereumjs-testrpc.svg)]()
-[![npm](https://img.shields.io/npm/dm/ethereumjs-testrpc.svg)]()
-[![Build Status](https://travis-ci.org/ethereumjs/testrpc.svg?branch=master)](https://travis-ci.org/ethereumjs/testrpc)
+[![npm](https://img.shields.io/npm/v/ganache-cli.svg)]()
+[![npm](https://img.shields.io/npm/dm/ganache-cli.svg)]()
+[![Build Status](https://travis-ci.org/trufflesuite/ganache-cli.svg?branch=master)](https://travis-ci.org/trufflesuite/ganache-cli)
 
-*NOTE: As of version `3.0.2`, `testrpc` requires at least `Node 6.9.1` to run - this is because the `ethereumjs-vm@2.0.1` dependency is now shipping using ES2015 language features.*
+<p align="center">
+  <img src="https://github.com/ethereumjs/testrpc/blob/ganache-cli/resources/icons/ganache-cli-128x128.png?raw=true">
```
Comment (tcoulter) [inline README.md:6]: Why monotone on the image? Sticking to Ganache-branded colors is the way to go. 

To give you more insight about our branding (also talk to @discriskandbisque): We're going to brand all of our products with an "accent color". Truffle's teal, Drizzle is pink and Ganache is orange. To go along with this branding I think you should keep the accent color here if you're going to include a branded image.

File: README.md [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
+</p>
 
-# Welcome to `testrpc`
+## Welcome to Ganache CLI
+Ganache CLI, part of the Truffle suite of Ethereum development tools, is the command line version of [Ganache](https://github.com/trufflesuite/ganache), your personal blockchain for Ethereum development.
 
-`testrpc` is a Node.js based Ethereum client for testing and development. It uses ethereumjs to simulate full client behavior and make developing Ethereum applications much faster. It also includes all popular RPC functions and features (like events) and can be run deterministically to make development a breeze.
+Ganache CLI uses ethereumjs to simulate full client behavior and make developing Ethereum applications much faster, easier, and safer. It also includes all popular RPC functions and features (like events) and can be run deterministically to make development a breeze.
```
Comment (tcoulter) [inline README.md]: I'd remove "much" if you're going to say if you're going to add "easier, and safer" (which is a good change).

File: README.md [hunk 3]
```diff
@@ -1,29 +1,36 @@
...
 
-# INSTALL
+### Looking for TestRPC?
 
-`testrpc` is written in Javascript and distributed as a Node package via `npm`. Make sure you have Node.js (>= v6.9.1) installed.
+If you came here expecting to find TestRPC, you're in the right place! Think of Ganache and `ganache-cli` as TestRPC evolved. We're starting off from the TestRPC codebase you already know and love, but adding heaps of new features, including a slick new UI, to make Ethereum platform development much safer, easier, and more enjoyable!
```
Comment (tcoulter) [inline README.md]: Branding and tone here is good, but I hesitate implying this is a new codebase (at least, that's how i read it). I'd go for something shorter and pump us up as benefactors:

> If you came here expecting to find the TestRPC, you're in the right place! Truffle has taken the TestRPC under its wing and made it part of the Truffle suite of tools. From now on you can expect better support along with tons of new features that help make Ethereum development safer, easier, and more enjoyable.

And perhaps follow this up with something that tells these readers about the replacement. e.g.,

> Just use `ganache-cli` on your command line instead of `testrpc`! 

Also, I switched "heaps" to "tons", because NZ. Feel free to switch it back, however. :)

File: README.md [hunk 4]
```diff
@@ -165,43 +172,41 @@ There’s also special non-standard methods that aren’t included within the or
 * `evm_increaseTime` : Jump forward in time. Takes one parameter, which is the amount of time to increase in seconds. Returns the total time adjustment, in seconds.
 * `evm_mine` : Force a block to be mined. Takes no parameters. Mines a block independent of whether or not mining is started or stopped.
 
-# Unsupported Methods
+### Unsupported Methods
 
 * `eth_compileSolidity`: If you'd like Solidity compilation in Javascript, please see the [solc-js project](https://github.com/ethereum/solc-js).
 
-# Docker
+### Docker
```
Comment (tcoulter) [inline README.md:179]: We actually have to register the `trufflesuite` docker org for this to work. Can you take this on? 

I'd almost not make it a requirement for the move. Instead remove the whole docker section and add it as a TODO for soon after the move.

File: README.md [hunk 5]
```diff
@@ -165,43 +172,41 @@ There’s also special non-standard methods that aren’t included within the or
...
+docker build -t trufflesuite/ganache-cli .
 ```
 
-# DEVELOPING
+### Contributing to Ganache CLI
 
 This is a distribution package where the core code is bundled to support browsers and reduce installation issues on all platforms. You can contribute to the core code via [ganache-core](https://github.com/trufflesuite/ganache-core).
 
-# TESTING
+### Testing Ganache CLI
```
Comment (tcoulter) [inline README.md]: This might be leftover from the move to `ganache-core`. Are there actually any tests in the CLI? Might consider removing this section if not.

File: README.md [hunk 6]
```diff
@@ -165,43 +172,41 @@ There’s also special non-standard methods that aren’t included within the or
...
 
 Run tests via:
 
 ```
 $ npm test
 ```
 
-# LICENSE
```
Comment (tcoulter) [inline README.md]: Why did you remove this section? At the very worst we should probably leave it and say "See LICENSE file".

File: cli.js [hunk 7]
```diff
@@ -136,8 +136,7 @@ if (options.fork) {
 
 var server = Ganache.server(options);
 
-//console.log("Ganache CLI v" + pkg.version);
-console.log("EthereumJS TestRPC v" + pkg.version + " (ganache-core: " + corepkg.version + ")");
+console.log("TruffleSuite Ganache CLI v" + pkg.version + " (ganache-core: " + corepkg.version + ")");
```
Comment (tcoulter) [inline cli.js]: I'd remove `TruffleSuite` here. "EthereumJS TestRPC" was the previous brand. This one is now (or at least, we want it to be known as) simply "Ganache CLI". This will help people ask: well, what is Ganache?

File: webpack/lib.webpack.config.js [hunk 8]
```diff
@@ -11,7 +11,7 @@ module.exports = {
   output: {
     path: outputDir,
     filename: outputFilename,
-    library: "TestRPC",
+    library: "ganache-cli",
```
Comment (tcoulter) [inline webpack/lib.webpack.config.js]: Here, `TestRPC` refers to the variable name output by the lib. e.g., this will generate code equivalent to:

```
var TestRPC = <... all of webpack's magic... >
```

Since `ganache-cli` isn't a valid variable name, we should choose something like `Ganache`. Also, CLI shouldn't be included since this is the name of the lib.

**Review by tcoulter — COMMENTED** [review]
Reviewed! Let me know if you have q's.

File: README.md (hunk 4 above)
Comment (benjamincburns) [reply README.md:179]: Will do this today.

Comment (benjamincburns) [reply README.md:179]: It really doesn't take long to register the org, so I'll do it after everything else is merged

File: README.md (hunk 1 above)
Comment (benjamincburns) [reply README.md:6]: Fixed.

File: README.md (hunk 3 above)
Comment (benjamincburns) [reply README.md]: I like your message better - switched.

File: README.md (hunk 5 above)
Comment (benjamincburns) [reply README.md]: There are not, but there really should be a smoke test or two...

File: README.md (hunk 6 above)
Comment (benjamincburns) [reply README.md]: I felt it was redundant - I feel the existence of the `LICENSE` file says this enough, as this is a very common FOSS project file.

File: cli.js (hunk 7 above)
Comment (benjamincburns) [reply cli.js]: Done.

File: webpack/lib.webpack.config.js (hunk 8 above)
Comment (benjamincburns) [reply webpack/lib.webpack.config.js]: Yeah, that was a careless find/replace on my part - fixed.

File: README.md (hunk 4 above)
Comment (benjamincburns) [reply README.md:179]: Will put back the "simplest way" bit later once the official image exists.

File: README.md (hunk 2 above)
Comment (benjamincburns) [reply README.md]: Done.
